manual/manual/refman/exten.etex

```diff
@@ -1539,6 +1539,28 @@ Some attributes are understood by the type-checker:
   of the string payload).  This is mostly useful for preprocessors which
   need to communicate warnings to the user.  This could also be used
   to mark explicitly some code location for further inspection.
+\item
+  ``ocaml.warn_on_literal_pattern'' or ``warn_on_literal_pattern'' annotate
+  constructors in type definition. The warning 53 is then emitted when this
+  constructor is pattern matched with a constant literal as argument.
```

Comment:
For other attributes documented in this list, the interest of using them is obvious. For this one you should give a justification -- fragility of `failwith` arguments, etc.